examples/jsm/controls/Controls.js

```diff
@@ -0,0 +1,32 @@
+import { EventDispatcher } from 'three';
+
+class Controls extends EventDispatcher {
+
+	constructor( object, domElement ) {
```

Comment:
@Mugen87 

`constructor( object, domElement = null )` maybe?

Right now it's `undefined` by default but subclasses set it to `null`...